num/quat/slerp_example_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright ©2022 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package quat_test
```

Comment:
I'm wondering with this doesn't belong in r3 rather than quat. I know that you are using this to link to r2, but I think that would be better done with a sentence in the package doc.

Reply:
I agree this is a better fit for `r3`. will move it over.